src/app/docs/architecture-design/key-management/page.tsx

```
@@ -9,265 +7,255 @@ import {
-        <h1 className="text-4xl font-bold text-gray-900 mb-6">智能密钥管理</h1>
-        <p className="text-xl text-gray-600 leading-relaxed">
-          GPT-Load
-          采用智能密钥管理机制，通过轮询负载均衡、自动故障检测和恢复机制，
-          确保服务的高可用性和稳定性。
-        </p>
+        <h1 className="text-4xl font-bold text-gray-900 mb-6">
+          {t("header.title")}
+        </h1>
+        <p
+          className="text-xl text-gray-600 leading-relaxed"
+          dangerouslySetInnerHTML={{ __html: t("header.description") }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)